website/src/Homepage.tsx

```diff
@@ -0,0 +1,309 @@
+justify-content: center;
+box-shadow: 0 1px 5px #0000001c;
+${props => props.gridStart ? `grid-column-start: ${props.gridStart};` : ""}
+${props => props.gridEnd ? `grid-column-end: ${props.gridEnd};` : ""}
+`;
+
+const FeatureAction = styled.div`
+position: absolute;
+bottom: 0;
+left:0;
+padding: 1.5rem;
+font-size: 1.1rem;
+box-sizing: border-box;
+border-top: 2px dashed #ffffff2f;
+width: 100%;
+text-align: right;
+grid-row-start: left-start;
+grid-row-end: end;
+z-index: 100;
+`;
+
+const FeatureChildren = styled.div`
+margin-bottom: 2.5rem;
+padding: 1.5rem;
+box-sizing: border-box;
+
+& p {
+  max-width: 60ch;
+}
+
+& img {
+  max-width: 100%;
+  
+}
+`;
+
+function Feature({ color, children, action, gridStart, gridEnd }: FeatureProps) {
+  return <FeatureContainer color={color} gridStart={gridStart} gridEnd={gridEnd}>
+    <FeatureChildren>{children}</FeatureChildren>
+    <FeatureAction>{action}</FeatureAction>
+  </FeatureContainer>
+}
+
+
+
+
+export default function Homepage() {
+  const navigate = useNavigate();
+  const menu = <MenuContainer>
+    <Link to="playground">Playground</Link>
+    <Link to="guide">Guide</Link>
+    <Link to="package-docs">Package docs</Link>
+    <a href="https://github.com/modmark-org/modmark"> GitHub</a>
+  </MenuContainer >;
+
+  return <Container>
+    <Hero>
+      <div>
+        {menu}
+        <Logo>
+          <img src="./logo.svg" alt="ModMark logo" />
+          <div>
+            <h1>ModMark</h1>
+            <h2>Modular Markup Language</h2>
+          </div>
+        </Logo>
+
+        <About>
+          <p>
+            ModMark is a modular markup language. It has a lightweight syntax akin to Markdown but also offers a lot more flexibility and expressive power. Import packages and use modules to add extra functionality in your document or even add support for a new output format.
+          </p>
+          <p>
+            ModMark comes bundled with multiple useful packages and can by default output both HTML and LaTeX documents. It is still a early protype but you can use it both on the web and in your terminal.
+          </p>
+        </About>
+        <ActionButton onClick={(_e) => navigate("/playground")}>Try it in your browser <FiPlay /></ActionButton>
+      </div>
+    </Hero >
+    <Features>
+      <Feature
+        action={<Link to="/guide">Getting started guide <FiBook /></Link>}
+        color="#816796"
+      >
+        <h1>
+          <span style={{ opacity: 0.4 }}>#</span> Familiar and <span style={{ opacity: 0.4 }}>**</span>powerful<span style={{ opacity: 0.4 }}>**</span>
+        </h1>
+        <p>
+          If you have used Markdown before you will quickly feel right at home.
+        </p>
+        <p>
+          Read the getting started guide to learn more about different ways to <CodeFont>**style** ==text==</CodeFont> and include other elements in your document like a <CodeFont>[image](./image.png)</CodeFont> or a <CodeFont>[link](https://modmark.org)</CodeFont>.
```

Comment:
It looks super cool, with all these codefonts and such, and I like that it looks kind of similar to markdown as an introduction. But, maybe, we shouldn't use `[image]` as an example since it doesn't really work in-line...